app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -72,6 +74,7 @@ class HomeFragment : Fragment() {
 
     private fun initTiles() = with (tileContainer) {
         val homeTiles = mutableListOf<HomeTile>()
+        homeTiles.addAll(HomeTilesManager.getCustomHomeTilesList(context))
```

Comment:
nit: if you don't mind that the `getCustomHomeTilesList` gets mutated, you can do
```kotlin
val homeTiles = Home...getCustomHomeTilesList(context).toMutableList()
```

i.e. this will mutate the existing list, rather than copying it into a new one.

Reply:
Not going to do this because CustomTileAccessor uses that list locally, without calling it, so it shouldn't be mutated.